components/identity-core/org.wso2.carbon.identity.core/src/main/java/org/wso2/carbon/identity/core/util/IdentityCoreConstants.java

```diff
@@ -42,6 +42,12 @@
     public static final String XML_DIGEST_ALGORITHM_SHA1 = "http://www.w3.org/2000/09/xmldsig#sha1";
 
     public static final String WEB_CONTEXT_ROOT = "WebContextRoot";
+
+    public static final String USER_ACCOUNT_DISABLED = " User account is disabled";
+
+    //UserCoreConstants class define the rest of the relevant error codes.
+    public static final String USER_ACCOUNT_DISABLED_ERROR_CODE = "17004";
```

Comment:
Constant was added as part of this PR. We can't keep it there as the constant is used by authentication framework as well, which in a upper rank in build order than identity.mgt package.